src/Tomcat/TomcatController.ts

```diff
@@ -60,43 +60,29 @@ export class TomcatController {
             throw (new Error(DialogMessage.noServer));
         }
 
-        const exist: boolean = await Utility.openFileIfExists(tomcatServer.getServerConfigPath());
-        if (!exist) {
-            throw (new Error(DialogMessage.noServerConfig));
+        const configFile: string = tomcatServer.getServerConfigPath();
+        if (!fse.pathExistsSync(configFile)) {
+            throw new Error(DialogMessage.noServerConfig);
         }
+        vscode.window.showTextDocument(vscode.Uri.file(configFile), { preview: false });
     }
 
     public async createTomcatServer(serverName: string, tomcatInstallPath: string): Promise<void> {
-        const catalinaBasePath: string = path.join(this._tomcat.getExtensionPath(), serverName);
-        const confPath: string = path.join(catalinaBasePath, 'conf');
-        const logPath: string = path.join(catalinaBasePath, 'logs');
-        const tempPath: string = path.join(catalinaBasePath, 'temp');
-        const webappsPath: string = path.join(catalinaBasePath, 'webapps');
-        const rootAppPath: string = path.join(webappsPath, 'ROOT');
-        const workPath: string = path.join(catalinaBasePath, 'work');
-
-        const serverConfigSrc: string = path.join(tomcatInstallPath, 'conf', 'server.xml');
-        const webConfigSrc: string = path.join(tomcatInstallPath, 'conf', 'web.xml');
-        const serverConfigTarget: string = path.join(catalinaBasePath, 'conf', 'server.xml');
-        const webConfigTarget: string = path.join(catalinaBasePath, 'conf', 'web.xml');
-        const rootPageTarget: string = path.join(rootAppPath, 'index.jsp');
-        let tomcatServer: TomcatServer;
-
-        await Utility.cleanAndCreateFolder(catalinaBasePath);
-        await Utility.cleanAndCreateFolder(confPath);
-        await Utility.cleanAndCreateFolder(logPath);
-        await Utility.cleanAndCreateFolder(tempPath);
-        await Utility.cleanAndCreateFolder(webappsPath);
-        await Utility.cleanAndCreateFolder(workPath);
-        await Utility.cleanAndCreateFolder(rootAppPath);
-        await fse.copy(serverConfigSrc, serverConfigTarget);
-        await fse.copy(webConfigSrc, webConfigTarget);
-
-        tomcatServer = new TomcatServer(serverName, tomcatInstallPath, this._tomcat.getExtensionPath());
+        const basePath: string = path.join(this._tomcat.getExtensionPath(), serverName);
```

Comment:
catalinaBasePath is special name/concept in tomcat. Better keep the origin name "catalinaBasePath"

Reply:
sure